Makefile

```diff
@@ -14,7 +14,7 @@ GOLANGCI_LINT_VERSION ?= v1.50.0
 KIND_VERSION ?= v0.16.0
 GOLICENSES_VERSION ?= v1.4.0
 # Current Operator version
-OPERATOR_VERSION ?= 0.24.0
+OPERATOR_VERSION ?= 0.25.0
```

Comment:
as discussed, see if `TAG` can be reused

Reply:
Yeap, I changed! :)

Reply:
What about the changes needed in `DEVEL.md`